io_context_test.go

```diff
@@ -9,34 +10,48 @@ import (
 )
 
 func TestIOContext(t *testing.T) {
-	var seeked bool
-	rb := []byte("read")
-	wb := []byte("write")
-	var written []byte
-	c, err := AllocIOContext(8, true, func(b []byte) (int, error) {
-		copy(b, rb)
-		return len(rb), nil
-	}, func(offset int64, whence int) (n int64, err error) {
-		seeked = true
-		return offset, nil
-	}, func(b []byte) (int, error) {
-		written = make([]byte, len(b))
-		copy(written, b)
-		return len(b), nil
+	t.Run("read write seek", func(t *testing.T) {
+		var seeked bool
+		rb := []byte("read")
+		wb := []byte("write")
+		var written []byte
+		c, err := AllocIOContext(8, true, func(b []byte) (int, error) {
+			copy(b, rb)
+			return len(rb), nil
+		}, func(offset int64, whence int) (n int64, err error) {
+			seeked = true
+			return offset, nil
+		}, func(b []byte) (int, error) {
+			written = make([]byte, len(b))
+			copy(written, b)
+			return len(b), nil
+		})
+		require.NoError(t, err)
+		defer c.Free()
+		b := make([]byte, 6)
+		n, err := c.Read(b)
+		require.NoError(t, err)
+		require.Equal(t, 4, n)
+		require.Equal(t, rb, b[:n])
+		_, err = c.Seek(2, 0)
+		require.NoError(t, err)
+		require.True(t, seeked)
+		c.Write(wb)
+		c.Flush()
+		require.Equal(t, wb, written)
+	})
+
+	t.Run("support go eof -> ffmpeg eof when reading", func(t *testing.T) {
```

Comment:
Could you rename the test run to `io.EOF is mapped to AVERROR_EOF when reading`

Reply:
Fixed!